base/bucket_gocb.go

```diff
@@ -147,6 +152,72 @@ func (bucket CouchbaseBucketGoCB) GetBucketCredentials() (username, password str
 	return bucket.spec.BucketName, password
 }
 
+func (bucket CouchbaseBucketGoCB) GetMetadataPurgeInterval() (int, error) {
```

Comment:
Switched RetrievePurgeInterval from public to private, and added some notes to make it clear that it's a helper function for GetMetadataPurgeInterval.